draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1698,9 +1704,11 @@ IANA is also requested to register the following identifiers in the AEAD Algorit
 
 # Test Vectors
 
+The following test vectors are also available in JSON format at [TEST-VECTORS]. In the JSON test vectors, byte string values are JSON strings containing the hex encoding of the byte strings.
```

Comment:
`[TEST-VECTORS]` should be `{{TEST-VECTORS}}` I believe.

I also wonder if the second sentence can be phrased better, but I see that you've taken it from RFC 9605.

Reply:
I did. But it can still be phrased better. What would you suggest?

Reply:
How about "In this format, byte strings are represented as JSON strings containing their hexadecimal encoding." ?

Reply:
> How about "In this format, byte strings are represented as JSON strings containing their hexadecimal encoding." ?

Yeah that's better.